Modelica/StateGraph.mo

```diff
@@ -1535,9 +1535,9 @@ buttons:
                        annotation (Placement(transformation(extent={{55,-50},{
                   65,-40}})));
       Real level(start=0,fixed=true) "Tank level in % of max height";
-      parameter Real A=1 "Ground area of tank in m^2";
-      parameter Real a=0.2 "Area of drain hole in m^2";
-      parameter Real hmax=1 "Max height of tank in m";
+      parameter Modelica.Units.SI.Area A=1 "Ground area of tank";
+      parameter Modelica.Units.SI.Area a=0.2 "Area of drain hole";
+      parameter Modelica.Units.SI.Height hmax=1 "Max height of tank";
```

Comment:
```suggestion
      parameter SI.Area A=1 "Ground area of tank";
      parameter SI.Area a=0.2 "Area of drain hole";
      parameter SI.Height hmax=1 "Max height of tank";
```

Reply:
Yes. IMO this makes physically sense.
The question is whether `level` shall be limited similarly. But maybe there could then be numerical problems.